### PR TITLE
Create book class and testing

### LINKED ISSUE #2: [1pt] Create a Book class
- All Book class properties visible in the diagram should be defined and set up in the constructor method.
- can_be_archived?() in the Book class
     - should override the method from the parent class
     - should return true if the parent's method returns true OR if cover_state equals "bad"
     - otherwise, it should return false
NB: Follow the TDD approach.

### PR DESCRIPTION
**Pull request by joskalenda**
# Create book class and add Test  ✅

### CODE REVIEW

File: classes/item.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ def initialize(publish_date, archived)
     @genre = nil
```
Comment (clintonjosephs) [inline classes/item.rb:12]: Hey @joskalenda, great job so far!. Kindly initialise @label to nil in the initialise block

File: classes/item.rb [hunk 2]
```diff
@@ -12,6 +12,11 @@ def initialize(publish_date, archived)
...
   end
 
+  def add_label(label)
```
Comment (clintonjosephs) [inline classes/item.rb:16]: I know this is probably an oversight but it's worth mentioning, the add_label method is a setter for the label instance variable.

You can rewrite it thus

`def label=(label) ........  end`

Good work on your implementation details

**Review by clintonjosephs — CHANGES_REQUESTED** [review]
Hey @joskalenda,

You have done a great job so far, However I pointed out just a few things that will make the project complete.

Feel free to reach out to me if you have any concerns or questions. Happy Coding ! 🤝

File: classes/item.rb (hunk 1 above)
Comment (joskalenda) [reply classes/item.rb:12]: 👍

File: classes/item.rb (hunk 2 above)
Comment (joskalenda) [reply classes/item.rb:16]: 👍

**Review by clintonjosephs — APPROVED** [review]
You have done a good job @joskalenda 👏 👏 , there is nothing else to say other than its time to merge it

**Pull request merged**